samples/debug/memfault/overlay-modem_trace_to_memfault.conf

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Continuously store modem traces to flash and upload them to Memfault upon a coredump
+
+# Memfault
+CONFIG_MEMFAULT_NCS_POST_MODEM_TRACE_ON_COREDUMP=y
+
+# nRF Modem lib trace
+CONFIG_FCB=y
+CONFIG_FLASH_MAP=y
+CONFIG_NRF_MODEM_LIB_TRACE=y
+CONFIG_NRF_MODEM_LIB_TRACE_LEVEL_OFF=y
```

Comment:
Maybe it would be good to have a comment here why the trace level should be off by default.

Reply:
added